File: java/FourClass.java
import java.util.Scanner;

public class FourClass {
    public static void main(String[] agrs){
        new FourClass();
    }
    public FourClass(){
        new Stu();
    }
    public class Stu{
        int roll;
        int age;
        String name;
        Scanner getin = new Scanner(System.in);
        public void get(){
            System.out.println("Enter your Name :");
            name=getin.nextLine();
            System.out.println("Enter your roll :");
            roll=getin.nextInt();
            System.out.println("Enter your age  :");
            age=getin.nextInt();
        }
        public void put(){
            System.out.println("\n\nName :" + name); 
            System.out.println("Roll :" + roll); 
            System.out.println("Age  :" + age);                      
        }
        public Stu(){
            get();
            put();
        }
    }
}

File: kotlin/fourclass.kt
import kotlin.io.*
class stu {
    var rol:Int =0
    var age:Int = 0
    var nam:String = ""
    fun get(){
        print("\nenter your name :")
        nam= readLine()!!
        print("enter your roll :")
        rol= readLine()!!.toInt()
        print("enter your age  :")
        age= readLine()!!.toInt()
    }
    fun put(){
        println("name : ${nam}")
        println("roll : ${rol}")
        println("age  : ${age}")
}
}
fun main() {
    var i:Int= 1
    print("Enter no of students :")
    var n:Int= readLine()!!.toInt()
    var b = Array(n) {stu()}
    for (stu in b){
        stu.get()
    }
    for(stu in b){
        println("\n\n ${i}")
        stu.put()
        i++
    }
}

  